model/code/registry-deprecated.yaml

```diff
@@ -8,35 +8,46 @@ groups:
       - id: code.function
         type: string
         stability: development
-        deprecated: Replaced by `code.function.name`
+        deprecated:
+          reason: renamed
+          renamed_to: code.function.name
```

Comment:
maybe better not to model as a rename since it also included the merging of code.namespace?